dem/installation/real-user-monitoring-installation.md

```diff
@@ -0,0 +1,59 @@
+---
+id: real-user-monitoring-installation
+title: Install Real User Monitoring
+---
+
+# Install Real User Monitoring
+
+# GDPR considerations
+
+<aside>
+🏆 Although DEM uses a cookie, **no consent is required.**
+</aside>
+
+The CNIL (French data protection authority) exempts certain cookies from requiring consent under these conditions:
+
+- they have a limited purpose such as measuring performance, detecting navigation issues, optimizing technical performance or usability...
+- they produce only anonymous statistics
+- they are not cross-referenced with other datasets
+- they are not transmitted to third parties
+- they do not enable tracking a user's browsing across other websites
+
+**DEM meets these conditions.**
+
+You can find CNIL's recommendations on this page: https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies-solutions-pour-les-outils-de-mesure-daudience
+
+# Find the tag to insert on my site
+
+Real User Monitoring (RUM) requires installing a JavaScript tag. The tag is available in the application by following this path:
+Go to Configuration, then *Integrations*:
+
+![image](../assets/installation/install-rum-1.png)
+
+You will then find the tag on that screen with a button to copy it easily:
+
+![image](../assets/installation/install-rum-2.png)
+
+This tag should be inserted into the site's HEAD section. The operation can be done manually by a developer, or alternatively **it can be added to a tag manager such as GTM by following the procedure below**:
```

Comment:
```suggestion
This tag should be inserted into the site's HEAD section. The operation can be done manually by a developer, or alternatively **it can be added to a tag manager such as GTM by following the procedure below**.
```